crates/ironrdp-displaycontrol/Cargo.toml

```diff
@@ -15,4 +15,5 @@ categories.workspace = true
 ironrdp-dvc.workspace = true
 ironrdp-pdu.workspace = true
 ironrdp-svc.workspace = true
+tokio = "1"
```

Comment:
issue: Hardcoding tokio here is a bit problematic. The most obvious problem is that this is not WASM friendly. And generally, we’ve been cautious to avoid tying the core crates to a specific async runtime (and async in general). This breaks this design. Here, I think we can use a trait or a callback instead.